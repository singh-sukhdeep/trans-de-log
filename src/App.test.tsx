import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';




describe('App component',()=>{

  test('renders learn react link', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders App Component',()=>{
    render(<App/>);
    const nameText=screen.getByText('Sukhdeep');
    expect(nameText).toBeInTheDocument();
  })
})