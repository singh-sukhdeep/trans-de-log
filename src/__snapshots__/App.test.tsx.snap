// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders learn react link 1`] = `
<div
  className="App"
>
  <h2>
    Sukhdeep
  </h2>
</div>
`;

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <h2>
    Sukhdeep
  </h2>
</div>
`;
